db_etl_homepage_graphs/grapher.py

```diff
@@ -96,6 +112,36 @@ def get_timeseries(date: str, metric: str):
     return True
 
 
+def get_value_50_plus(item):
+    ''' Get the values form the element (dict) in a list
+        where its key 'age' is '50+' and put them in new dict keys:
+        - cumPeopleVaccinatedAutumn22ByVaccinationDate
+        - cumVaccinationAutumn22UptakeByVaccinationDatePercentage
+    '''
```

Comment:
Could we use the reST style of docstring?

"""
This is a reST style.

:param param1: this is a first param
:param param2: this is a second param
:returns: this is a description of what is returned
:raises keyError: raises an exception
"""